File: C#/Урок 038/Задача 13.cs
class Program
{
    static void Main()
    {
        const decimal DEPOSIT_FACTOR = 0.05M;

        System.Console.Write("Enter initial sum: ");
        decimal initialSum = System.Convert.ToDecimal(System.Console.ReadLine());

        System.Console.Write("Enter years: ");
        int years = System.Convert.ToInt32(System.Console.ReadLine());

        decimal sum = initialSum;

        int currentYear = 1;
        while (currentYear <= years)
        {
            System.Console.Write($"In {currentYear} year(s) sum will be: {sum:C} + ({sum:C} + {sum:C} * {DEPOSIT_FACTOR}) = ");
            sum += sum + sum * DEPOSIT_FACTOR;
            System.Console.WriteLine($"{sum:C}");

            currentYear++;
        }
    }
}

File: C++/Урок 032 - while/Задача 13.cpp
#include <iostream>

int main()
{
    std::cout << "Enter deposit: $";
    double deposit;
    std::cin >> deposit;

    std::cout << "Enter amount of years: ";
    int years;
    std::cin >> years;

    const double INITIAL_DEPOSIT = deposit;
    const double INCREASE_FACTOR = 0.05;

    int year = 1;
    while (year <= years)
    {
        std::cout << "In " << year << " year(s) deposit will be:\n";
        std::cout << deposit << " + (" << deposit << " * " << INCREASE_FACTOR << ")";

        deposit += deposit + deposit * INCREASE_FACTOR;
        std::cout << " = $" << deposit << "\n\n";

        year++;
    }
    
    std::cout << "The final deposit will be $" << deposit << std::endl;

    return 0;
}